modules/UI/videolayout/SmallVideo.js

```diff
@@ -99,6 +101,30 @@ function SmallVideo(VideoLayout) {
      */
     this._popoverIsHovered = false;
 
+    /**
+     * Whether or not the connection indicator should be displayed.
+     *
+     * @private
+     * @type {boolean}
+     */
+    this._showConnectionIndicator = true;
+
+    /**
+     * Whether or not the dominant speaker indicator should be displayed.
+     *
+     * @private
+     * @type {boolean}
+     */
+    this._showDominantSpeaker = false;
+
+    /**
+     * Whether or not the raised hand indicator should be displayed.
+     *
+     * @private
+     * @type {boolean}
+     */
+    this._showRaisedHand = false;
```

Comment:
With React, the indicators will all be re-rendered so cache the display values for dominant speaker and raised hand andconnection indicator so when the visibility of only one indicators gets updated, forcing a re-render of all three with the current dom hierarchy, the rest will keep their state.